htdocs/compta/bank/categ.php

```diff
@@ -118,44 +89,28 @@
 print '<td>'.$langs->trans("Ref").'</td><td colspan="2">'.$langs->trans("Label").'</td>';
 print "</tr>\n";
 
-$sql = "SELECT rowid, label";
```

Comment:
When a list of record is output as a list into a page, we must keep using the structure with request inside the page and not used the fetchAll.
The reason is that the fetchAll is not ready for a lot of new coming feature like "pagination", dynamic limit of number of records, filtering, extrafields management. All of this is ok with old structur, so could you keep the list with loop inside page instead of fetchAll for this part of code ?